src/components/form/select/index.js

```diff
@@ -10,8 +10,8 @@ const Select = ({ register, options, name, error, placeholder, className = '', .
         {placeholder}
       </option>
       {options.map((value, i) => (
-        <option key={i} value={value}>
-          {value.label ? value.label : value}
+        <option key={i} value={value.topic ? value.topic.id : value}>
```

Comment:
The `Select` component defined in this file should be generic. It shouldn't check for a `topic` object in the `options` prop. Instead, you should pass a proper `options` prop for the `Select` component at `src/pages/createTarget/index.js`.

I would revert this change. Check my comment above.